File: HA1_4.java
import java.util.*;

class Number
{
    public boolean ChkDivisible(int iNo)
    {
        if((iNo %5)==0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

public class HA1_4
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter a number");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        {
            boolean bRet = nobj.ChkDivisible(iValue);

            if(bRet == true)
            {
                System.out.println("It is divisible by 5");
            }
            else
            {
                System.out.println("It is not divisble by 5");
            }
        }
        sobj.close();
    }
}

File: HA1_4.c
//Accept onr number and check wheather it is divisible by 5 or not
#include<stdio.h>
typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL ChkDivisible(int iNo)
{
    if ((iNo % 5)==0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter a number \n");
    scanf("%d", &iValue);

    bRet = ChkDivisible(iValue);

    if(bRet == TRUE)
    {
        printf("It is divisible by 5");
    }
    else
    {
        printf("It is not divisible by 5");
    }
    return 0;
}